dev-proxy/dev-proxy.csproj

```diff
@@ -50,27 +50,32 @@
     <None Update="devproxyrc.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="devproxy-errors.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="toggle-proxy.sh">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
     <None Update="picture.jpg">
       <CopyToOutputDirectory>Never</CopyToOutputDirectory>
     </None>
-    <None Update="responses.json">
+    <None Update="trust-cert.sh">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="trust-cert.sh">
+    <None Update="presets\microsoft-graph.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="responses.sample.json">
+    <None Update="presets\microsoft-graph-rate-limiting.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="presets\microsoft-graph.json">
+    <None Update="presets\microsoft-365.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="presets\microsoft-graph-rate-limiting.json">
+    <None Update="presets\microsoft-365-errors.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties default_1json__JsonSchema="https://json.schemastore.org/global.json" devproxyrc_1json__JsonSchema="https://json.schemastore.org/global.json" errors_1json__JsonSchema="https://json.schemastore.org/global.json" /></VisualStudio></ProjectExtensions>
```

Comment:
Do we need this?

Reply:
No, this snuck in, good catch. I'll remove.